src/main/java/uk/gov/companieshouse/charges/data/repository/ChargesRepository.java

```diff
@@ -9,7 +9,10 @@
 @Repository
 public interface ChargesRepository extends MongoRepository<ChargesDocument, String> {
 
+    @Query("{'company_number': ?0, '_id': ?1 }")
+    List<ChargesDocument> findChargeDetails(String companyNumber, String chargeId);
```

Comment:
Can we please change this to Option<ChargesDocument> instead, for companyNumber and chargeId combination, it should not have more than 1 entry

Reply:
Updated to Optional